lblod/app-subsidiepunt
# Add fusie bestuurseenheden & create mock accounts

## ID
DGS-415

## Description
New bestuurseenheden for fusies. Migrations come from loket. Only data needed for subsidiepunt has been imported. 

ref: https://github.com/lblod/app-digitaal-loket/pull/611/files

## Type of change


## How to setup
Setup app as usual

## How to test
Make sure migrations have run, check if all new fusie accounts are available thorugh mock login and you can login with them
